src/pages/resultsView/enrichments/GeneBarPlot.tsx

```diff
@@ -132,7 +141,7 @@ export default class GeneBarPlot extends React.Component<IGeneBarPlotProps, {}>
 
     @computed get label() {
         const label = this._label || this.defaultOption.label;
-        return `${label.charAt(0).toUpperCase()}${label.substring(1)}`;
+        return `Showing ${label.charAt(0).toLowerCase()}${label.substring(1)}`;
```

Comment:
I got rid of Showing b/c it was taking up too much space.  I also capitalized the first letter.